src/renderers/common/Bindings.js

```diff
@@ -162,6 +162,16 @@ class Bindings extends DataMap {
 
 		const bindings = this.nodes.getForCompute( computeNode ).bindings;
 
+		if ( this.backend.isWebGPUBackend ) {
+
+			for ( const bindGroup of bindings ) {
+
+				this.backend.deleteBindGroupData( bindGroup );
```

Comment:
Please refactor this part and remove the `if ( this.backend.isWebGPUBackend ) {` check. The idea is to define `deleteBindGroupData()` as an abstract method in `Backend` and don't provide an implementation for the WebGL backend.